lib/modules/manager/renovate-config-presets/index.ts

```diff
@@ -8,7 +8,7 @@ export { extractPackageFile } from './extract';
 export const url = '../../../config-presets.md';
 
 export const defaultConfig = {
-  managerFilePatterns: configFileNames.filter(
+  managerFilePatterns: getConfigFileNames().filter(
```

Comment:
same here global config isn't set when this module scope code is executed

Reply:
What should we do here, then?

Loosen the patterns or add a comment in the docs asking users to add filepatterns to this manager if they want to use it with custom config filenames.